apps/definition_deadletter/.gitignore

```diff
@@ -20,5 +20,5 @@ erl_crash.dump
 *.ez
 
 # Ignore package tarball (built via "mix hex.build").
-broadway_configurator-*.tar
+definition_deadletter-*.tar
```

Comment:
Is this file an actual rename or is git just confused by you guys removing one app and adding another?

Reply:
git is just confused